### PR TITLE
Restyle dashboard and czar player

### PR DESCRIPTION
**Pull request by Del-sama**
#### What does this PR do?
* Redesigns ```dashboard``` and ```czar player```
#### Description of Task to be completed?
* It adds margin-left to ```friends``` option in dashboard
* It removes background color nd changes position of czar
#### How should this be manually tested?
* By running ```npm start```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: public/js/controllers/game.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ angular.module('mean.system')
       $scope.abandonGame = () => {
         game.leaveGame();
         gameTour.cancelTour();
-        $location.path('/');
+        window.location.href = '../';
```
Comment (ooduye) [inline public/js/controllers/game.js:152]: Why this change?

**Comment by ooduye** [comment]
Why is the PR not building?

File: public/js/controllers/game.js (hunk 1 above)
Comment (Del-sama) [reply public/js/controllers/game.js:152]: the removed one takes u home but maintains url of game but the new one fixes the issue and carries out a full redirect.
